resources/js/src/app/plugins/lozad.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Detect IE browser
```

Comment:
Warum wird die lib nicht mehr über die node_modules eingebunden? Bitte eine kurze Erklärung in das Kommentar ergänzen.

Reply:
Wir sind auf eine Limitierung von Lozad gestoßen. Wir mussten eine Klasse direkt auf das zu generierende img Tag dynamisch setzen. 
Lukas und ich haben über Lösungen diskutiert und uns entschieden, Lozad direkt zu integrieren und für unsere Zwecke anzupassen.

Reply:
> Bitte eine kurze Erklärung in das Kommentar ergänzen.

Reply:
Done.